R/ArchiveTuning.R

```diff
@@ -1,14 +1,66 @@
-#' @title Logging object for objective function evaluations
+#' @title Logging Object for Evaluated Hyperparameter Configurations
 #'
 #' @description
-#' Container around a [data.table::data.table()] which stores all performed
-#' function calls of the Objective and the associated [mlr3::BenchmarkResult].
+#' Container around a [data.table::data.table()] which stores all evaluated
+#' hyperparameter configurations and performance scores.
 #'
-#' `$benchmark_result` stores a [mlr3::BenchmarkResult] which contains the
-#' [mlr3::ResampleResult] of all performed function calls. The
-#' [mlr3::BenchmarkResult] is connected to the [data.table::data.table] via the
-#' `uhash` column.
+#' @section Data structure:
 #'
+#' The table (`$data`) has the following columns:
+#'
+#' * One column for each hyperparameter of the search space (`$search_space`).
+#' * One column for each performance measure (`$codomain`).
+#' * `x_domain` (`list()`)\cr
+#'   Lists of transformed hyperparameter values that are passed to the learner.
+#' * `runtime` (`numeric(1)`)\cr
+#'   Sum of all training and predict times logged per evaluation.
```

Comment:
Could we say that this is CPU Time? So if we have parallelization enabled the "runtime" can be bigger than the time diff between two evals, right?
Is there a potential overhead that is ignored here (e.g. time that is neither training nor predict?) @mllg